admin/src/react-components/account-edit-toolbar.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { Toolbar, SaveButton } from "react-admin";
+import { withStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import { Dialog, DialogContent, DialogContentText, DialogActions } from "@material-ui/core";
+
+const accountEditToolbarStyles = {
+  spaceBetween: { justifyContent: "space-between" },
+  dialogActions: { padding: "0 5px" }
+};
+
+const deleteStates = {
```

Comment:
when working if consts like this I usually like to use capitals 

const deleteStates = {
  Confirming: "confirming",
  Deleting: "deleting",
  Succeeded: "succeeded",
  Failed: "failed"
};

and then in the component i'd destruct that { Confirming, Deleting, Succeeded, Failed } = deleteStates

I like this because you can see where the consts come from in the top of the file "deleteStates" but then you can just throw around Confirm without dot notation. And then when i see a capital like "Confirming" then I know it's a const and not a var.